src/common/store/gameFormats.js

```diff
@@ -1,10 +1,19 @@
 import { cloneDeep, shuffle } from 'lodash';
 import * as seededRNG from 'seed-random';
 
+import { DECK_SIZE } from '../constants';
 import { triggerSound } from '../util/game';
 
 import defaultState, { bluePlayerState, orangePlayerState } from './defaultGameState';
 
+function deckHasNCards(deck, num) {
+  return deck.cardIds.length === num;
+}
+
+function deckHasOnlyBuiltinCards(deck) {
+  return deck.cardIds.every(cardId => cardId.startsWith('builtin/'));
```

Comment:
Is there anything stopping creating custom cards under built-in/?

Reply:
Custom cars have a random ID, so it's unlikely that it would start with `builtin/` 😛

Reply:
So you're saying there's a chance 😉

Reply:
Yeah, I can try to come up with a better approach for this in a separate PR.